script/docker_push

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+docker push homeassistant/home-assistant
```

Comment:
How is this image supposed to be pushed when it is not built anywhere?
Also you would need to push the release tag too.